front_end/html/profile_instructor.html

```diff
@@ -111,17 +111,17 @@ <h3>Instructor Menu for {{ course['title'] }}</h3>
         {% end %}>
 
             <h2>Current instructors:</h2>
-            
+
             {% if instructors %}
                 <ul>
                 {% for user in instructors %}
-                    <li>{{ user }} <a onclick="showRemoveInstructorModal({{ user }})">remove</a></li>
+                    <li>{{ user }} <a onclick="showRemoveInstructorModal('{{ user }}')">remove</a></li>
```

Comment:
This fixes a bug where admin/instructor roles were unable to to remove instructors and assistants from a class.